pychatlib/client/client.py

```diff
@@ -20,13 +20,28 @@ def __init__(self):
             "slid in!",
             "showed up!",
             "joined the party!",
+            "has found the chat!",
+            "has found a new cake for us!",
+            "has found us! Quick! Hide the cake!",
+            ", do you have ZirCoins?",
+            "might be the impostor"
         ]
         self.LEAVE_MESSAGES = [
             "left the chat",
             "has left",
             "just left",
             "has exited",
             "flew away!",
+            "has found the exit",
```

Comment:
Change "left to touch the grass" to "left to touch grass"
Change "was exposed by ALT + F4" to "pressed ALT  + F4"